prime-router/metadata/hl7_mapping/resources/URI/HD.yml

```diff
@@ -0,0 +1,19 @@
+elements:
+
+  - name: valueHD1
+    condition: '%resource.extension(%`rsext-cx-4-1-extension`).exists()'
```

Comment:
So this is a little confusing to me, since this doesn't seem to actually  be able to go from any URI -> HD.  URI is also not a resource

Reply:
With more generic extension names, this might make a little more sense. Moved it to `datatypes` as well. I added an element to populate HD1 with the value in the absence of the extensions; that seemed harmless enough and makes this more broadly applicable.

Reply:
Changes make sense to me!